project/api/models/Question.py

```diff
@@ -7,9 +7,9 @@
 class Question(models.Model):
     prompt = models.CharField(max_length=255)
     quiz = models.ForeignKey(Quiz, on_delete=models.CASCADE)
-
-    # def get_correct_answers(self):
-    #     pass
+    name = models.CharField(max_length=255, default='')
```

Comment:
Why do we need the name field again?

Reply:
So it can have a title somewhere instead of having to print out the description.